### ISSUE TITLE
Admin General Settings Panel - Ensure all controls are accessible by keyboard

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#241]
<img width="1679" alt="Screen Shot 2020-04-21 at 1 17 57 PM" src="https://user-images.githubusercontent.com/31868510/79893991-d2ba0a80-83d2-11ea-9816-9985b7e076ae.png">

<img width="1680" alt="Screen Shot 2020-04-21 at 1 23 02 PM" src="https://user-images.githubusercontent.com/31868510/79894717-de5a0100-83d3-11ea-8e4a-6858b8f89c8d.png">

<img width="1679" alt="Screen Shot 2020-04-21 at 1 24 38 PM" src="https://user-images.githubusercontent.com/31868510/79894739-e5810f00-83d3-11ea-8d16-c0b6224b5357.png">

Indicated controls are not accessible by keyboard navigation alone. You can read about this WCAG success criterion here:

https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html
